plugins/modules/cloud/misc/proxmox_user_info.py

```diff
@@ -224,7 +224,7 @@ def main():
     module = AnsibleModule(
         argument_spec=module_args,
         required_one_of=[('api_password', 'api_token_id')],
-        required_together=[('api_token_id', 'api_token_secret')],
+        required_together=[('api_token_id', 'api_token_secret'), ('user', 'domain')],
```

Comment:
This part is not correct, `domain` can be used independently of `user` as shown in the module documentation. Perhaps a `required_if` or `required_by` could do the trick though?

Reply:
Thanks for the review @Aversiste ! Will work on that tonight (GMT+13).

Reply:
Rolled back this one. Thanks again for the review.